tests/storyshot.test.ts

```diff
@@ -0,0 +1,15 @@
+import initStoryshots from '@storybook/addon-storyshots';
+import { render } from '@testing-library/react';
+import { resolve } from 'path';
+
+jest.mock('uuid', () => {
```

Comment:
why do we need this mock?

Reply:
We need this mock because uuid generated each time with the test rendering is different, so it is recognized by storyshot add-on as a code implementation change.